src/test/java/Homework16.java

```diff
@@ -0,0 +1,59 @@
+import org.checkerframework.checker.units.qual.C;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 {
+
```

Comment:
you can put all the variables in the class so as not to write them in every method of this class url.by etc